src/bsd.c

```diff
@@ -1202,6 +1187,7 @@ handle_curl_msg(CURLMsg *msg)
         }
       }
       if (resp->too_big) {
+        pe_regs_set_int(resp->pe_regs, PE_REGS_Q, "http-overflow", 1);
```

Comment:
Should I just set this into %1? 
Right now, this is being set into a q-register (and 0 never being set on success, which is bad, as it can cause false-positives) because status & content-type are being set into q-regs on completion.

Reply:
Input still needed on this @talvo